extension.ts

```diff
@@ -217,6 +231,13 @@ async function handleKeyEvent(key: string): Promise<void> {
   });
 }
 
+function handleContentChangedFromDisk(document : vscode.TextDocument) : void {
+  _.filter(modeHandlerToEditorIdentity, (modeHandler) => modeHandler.fileName === document.fileName)
+    .forEach((modeHandler) => {
```

Comment:
<img width="706" alt="screen shot 2016-09-03 at 3 44 12 am" src="https://cloud.githubusercontent.com/assets/2920178/18220581/cc7d90de-7188-11e6-80f2-0666ec97b447.png">
@jpoon  I saw other places are always using brackets even for a single argument, that's why i added those brackets 😄